test/blackbox-tests/test-cases/menhir/run.t

```diff
@@ -75,8 +75,8 @@ Reproduction case for #1781, only the .ml and .mli should be promoted:
   $ dune build @all --root promote
   Entering directory 'promote'
   $ ls -1 promote/_build/default | grep mock
```

Comment:
maybe we need a LANG=C or something here. Actually, maybe we should set LANG=C in the cram test executable

Reply:
I pushed a commit that does this but it doesn't seem to have an effect.

Reply:
Travis still seems to prefer the old order. Shall we just sort the results explicitly?

Reply:
Yh, I checked and we are already doing `| sort` explicitly in a few other tests